web/public/mgmt/config.yml

```diff
@@ -2522,6 +2522,54 @@ collections:
           - { label: "school/course", value: "school-course" }
           - { label: "object or vehicle", value: "object-vehicle" }
 
+  - name: "raffle-bingo-steps"
+    label: "Raffle Bingo Steps (Navigator with Webflow mappings)"
+    folder: "content/src/roadmaps/raffle-bingo-steps"
+    summary: "{{displayname}} - {{name}}"
+    identifier_field: "{{filename}}"
+    slug: "{{fields.filename}}"
+    delete: true
+    sortable_fields: ["id", "name", "{{urlSlug}}", "industryId"]
+    create: true
+    editor:
+      preview: true
```

Comment:
If you want to make this feel a little more complete you could implement the CMS preview, but not necessary for the ticket to be complete.

<img width="1510" alt="Screenshot 2024-09-30 at 12 00 39 PM" src="https://github.com/user-attachments/assets/aa26619d-8f1c-4579-a26c-ab264233f083">